.github/scripts/rebase-finish.sh

```diff
@@ -4,13 +4,20 @@ source `dirname ${BASH_SOURCE[0]}`/config.sh
 
 REBASE_BRANCH=$1
 ORIGIN_BRANCH=$2
+BACKUP_BRANCH=$3
 
 git fetch --all --prune
 
+# Check if $ORIGIN_BRANCH didn't change.
+if [ "$(git rev-parse origin/$BACKUP_BRANCH)" != "$(git rev-parse origin/$ORIGIN_BRANCH)" ]; then
+    echo "$ORIGIN_BRANCH has changed!"
+    exit 1
+fi
+
 # Check if $REBASE_BRANCH exists.
 if ! git show-ref --verify --quiet refs/remotes/origin/$REBASE_BRANCH; then
-  echo "$REBASE_BRANCH does not exist!"
-  exit 1
+    echo "$REBASE_BRANCH does not exist!"
+    exit 1
```

Comment:
whitespacing

Reply:
Yes, this fixes the formatting to the convention.